octobot_trading/exchanges/exchange_simulator.py

```diff
@@ -179,6 +184,15 @@ def get_trade_fee(self, symbol, order_type, quantity, price,
             FeePropertyColumns.COST.value: cost
         }
 
+    def get_traded_pairs(self, importer):
+        return set(get_available_symbols(importer)) & \
+               set(self.exchange_manager.exchange_config.traded_symbol_pairs)
+
+    def get_time_frames(self, importer):
+        return sort_time_frames(set(get_available_time_frames(importer)) &
```

Comment:
Same here is `& set(self.exchange_manager.exchange_config.traded_time_frames)` necessary ?

Reply:
This one is because we want timeframes that are available in data file and that are required by the strategy

Reply:
:+1: